src/giph

```diff
@@ -96,12 +99,33 @@ function log_info() {
   log "\033[0;36mINFO:\033[0m $1" 0 "${2:-true}"
 }
 
+# send a notification - (message:string, urgency:string, durationms:int)
```

Comment:
The urgency argument isn't really used anywhere, so it's dead code, please just use urgency 1 instead. This can be implemented if it's used somewhere.